Python.gitignore

```diff
@@ -43,6 +43,7 @@ htmlcov/
 nosetests.xml
 coverage.xml
 *,cover
+.hypothesis/
```

Comment:
The problem is that .hypothesis directory may occurs not only at the project root, but in project subdirectories as well (like tests/.hypothesis). This depends on how you run the tests. To ignore these cases you need to remove trailing slash. FYI @DRMacIver